CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -31,7 +31,10 @@ open class CenteredCollectionViewFlowLayout: UICollectionViewFlowLayout {
 			return itemSize.width + minimumLineSpacing
 		case .vertical:
 			return itemSize.height + minimumLineSpacing
-		}
+            
+        default:
+            return 0
+        }
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?